src/video-player.js

```diff
@@ -606,6 +619,65 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     return this.videojs.cloudinary.cloudinaryConfig(config);
   }
 
+  _setGoBackButton() {
+    const button = createElement('button', { 'class': 'go-back-button' }, 'Go back');
+
+    button.addEventListener('click', () => {
+      this.goBack && this.goBack();
+    }, false);
+
+    const tracksContainer = createElement('div', { 'class': TRACKERS_CONTAINER_CLASS_NAME }, button);
+    setTrackersContainer(this.videojs, tracksContainer);
+  }
+
+  addTrackers(tracksData, trackersOptions) {
+    const currentSourceUrl = this.currentSourceUrl();
+
+    this.goBack = () => {
+      if (this.isZoomed) {
+        this.isZoomed = false;
+        this.source(currentSourceUrl).play();
+        this._addTrackersItems(tracksData, trackersOptions);
+      }
+    };
+
+    this._addTrackersItems(tracksData, trackersOptions);
+  }
+
+  _addTrackersItems(trackersData, trackersOptions) {
+    const trackerItems = trackersData.map((item, index) => {
+      return getTrackerItem(item, (event) => {
+        this.isZoomed = true;
+        this._setGoBackButton();
+        trackersOptions && trackersOptions.onClick({ item, index, event });
+      });
+    });
+
+    const tracksContainer = createElement('div', { 'class': TRACKERS_CONTAINER_CLASS_NAME }, trackerItems);
+
+    setTrackersContainer(this.videojs, tracksContainer);
+  }
+
+  addCueListener() {
```

Comment:
״הכנה למזגן״

Reply:
This is why you removed this rule right?
"no-unused-expressions": "off"
why is the unused code part of this pr?